src/popup/pages/Address.vue

```diff
@@ -35,10 +37,16 @@ export default defineComponent({
 
     onMounted(async () => {
       try {
+        setLoaderVisible(true);
         await openModal(MODAL_CONFIRM_CONNECT, {
           access: [POPUP_CONNECT_ADDRESS_PERMISSION],
           app: app.value,
         });
+
+        /**
+         * Sdk should be initialized in order to get a current networkId.
+         */
```

Comment:
No need to use DocBlock here.

```suggestion
        // Sdk should be initialized in order to get a current networkId.
```